src/wasm.h

```diff
@@ -227,6 +227,15 @@ class Literal {
       default: abort();
     }
   }
+  
+  int32_t geti64Low() {
```

Comment:
We can put the methods on the I64Literal-class, but then we need a way to convert a `Literal` instance(returned by `callExport()`) to `I64Literal` in order to call the methods. We could add a constructor to `I64Literal` that takes a `Literal` as argument, but that feels somewhat convoluted. Making `geti64Low()` & `geti64High()` static methods on `I64Literal` is the best I can think of but it seems clumsy, too.